public/vendor/firebase-7.12.0/firebase-auth.js

```diff
@@ -0,0 +1,2 @@
h(t){throw new I("argument-error","Either an instance of firebase.auth.Auth must be passed as an argument to the firebase.auth.PhoneAuthProvider constructor, or the default firebase App instance must be initialized via firebase.initializeApp().")}Ki(this,{providerId:"phone",isOAuthProvider:!1})}function Oo(t,n){if(!t)throw new I("missing-verification-id");if(!n)throw new I("missing-verification-code");return new ko({bb:t,ab:n})}function Ro(t){if(t.temporaryProof&&t.phoneNumber)return new ko({Ja:t.temporaryProof,da:t.phoneNumber});var n=t&&t.providerId;if(!n||"password"===n)return null;var e=t&&t.oauthAccessToken,i=t&&t.oauthTokenSecret,r=t&&t.nonce,o=t&&t.oauthIdToken,a=t&&t.pendingToken;try{switch(n){case"google.com":return go(o,e);case"facebook.com":return fo(e);case"github.com":return vo(e);case"twitter.com":return yo(e,i);default:return e||i||o||a?a?0==n.indexOf("saml.")?new eo(n,a):new oo(n,{pendingToken:a,idToken:t.oauthIdToken,accessToken:t.oauthAccessToken},n):new ho(n).credential({idToken:o,accessToken:e,rawNonce:r}):null}}catch(t){return null}}function Po(t){if(!t.isOAuthProvider)throw new I("invalid-oauth-provider")}function Co(t,n,e,i,r,o,a){if(this.c=t,this.b=n||null,this.g=e||null,this.f=i||null,this.i=o||null,this.h=a||null,this.a=r||null,!this.g&&!this.a)throw new I("invalid-auth-event");if(this.g&&this.a)throw new I("invalid-auth-event");if(this.g&&!this.f)throw new I("invalid-auth-event")}function Do(t){return(t=t||{}).type?new Co(t.type,t.eventId,t.urlResponse,t.sessionId,t.error&&T(t.error),t.postBody,t.tenantId):null}function Lo(){this.b=null,this.a=[]}$r.prototype.Fa=function(){return this.a?En(this.a):En(this.b)},$r.prototype.v=function(){return this.type==Zr?{multiFactorSession:{idToken:this.a}}:{multiFactorSession:{pendingCredential:this.b}}},to.prototype.ia=function(){},to.prototype.b=function(){},to.prototype.c=function(){},to.prototype.v=function(){},eo.prototype.ia=function(t){return hs(t,io(this))},eo.prototype.b=function(t,n){var e=io(this);return e.idToken=n,ls(t,e)},eo.prototype.c=function(t,n){return no(fs(t,io(this)),n)},eo.prototype.v=function(){return{providerId:this.providerId,signInMethod:this.signInMethod,pendingToken:this.a}},oo.prototype.ia=function(t){return hs(t,ao(this))},oo.prototype.b=function(t,n){var e=ao(this);return e.idToken=n,ls(t,e)},oo.prototype.c=function(t,n){return no(fs(t,ao(this)),n)},oo.prototype.v=function(){var t={providerId:this.providerId,signInMethod:this.signInMethod};return this.idToken&&(t.oauthIdToken=this.idToken),this.accessToken&&(t.oauthAccessToken=this.accessToken),this.secret&&(t.oauthTokenSecret=this.secret),this.nonce&&(t.nonce=this.nonce),this.a&&(t.pendingToken=this.a),t},uo.prototype.Ia=function(t){return this.Fb=ct(t),this},w(co,uo),w(ho,uo),ho.prototype.Aa=function(t){return K(this.a,t)||this.a.push(t),this},ho.prototype.Nb=function(){return W(this.a)},ho.prototype.credential=function(t,n){var e;if(!(e=v(t)?{idToken:t.idToken||null,accessToken:t.accessToken||null,n
```

Comment:
add newline

Reply:
@IsaackMwenda, On the editor it is formatted. I think it appears that because its too long.